lib/dispatch.go

```diff
@@ -411,3 +416,50 @@ func dispatchReturnError(got interface{}, err error) error {
 	}
 	return err
 }
+
+// normalizeInputParams will look at each field of the params, and modify filepaths so that
+// they are absolute paths, making them safe to send across RPC to another process
+func normalizeInputParams(param interface{}) interface{} {
```

Comment:
What do you think about moving these to `params.go` in this same package? While this is only called within dispatch, it feels like a separate concept.

Reply:
Sounds good to me, agreed it's somewhat removed from dispatch.